### DISCUSSION TITLE
Deep linking not working

### DISCUSSION THREAD

**Discussion by Muddassir-Farooq-official** [flet-dev/flet#4188] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

when I open this link in browser https://mydomain.com then my app doesn't open automatically I add all information in py project.toml already

### Code sample

<details open><summary>Code</summary>

```python
[Paste your code here]
```

</details>

### To reproduce

I want that when I paste this URL in browser it should automatically open my application https://mydomain.com

### Screenshots / Videos

<details open>
<summary>Captures</summary>

[Upload media here]

</details>

### Operating System

Windows

### Operating system details

Windows 10

### Flet version

0.25.0.dev3519

### Regression

No, it isn't

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by FeodorFitsner** [comment]
It's more than just configuring an intent in an Android page.

Take a look at this article: https://codewithandrea.com/articles/flutter-deep-links/
There must be server verification too.

I wasn't able to make deep linking work on Android with `https` scheme, but it works pretty well with a custom theme, e.g. `flet` we use in a Flet app published to Google Store.

Also, there is a trick to make your app open by clicking a link with a custom theme. If you click `flet://something` on a web page it will give you a "Not found" response, but if you trigger URL redirection via JavaScript it works:

```
<!DOCTYPE html>
<html>

<head>
    <title>Redirect Page</title>
    <meta charset="UTF-8" />
    <style>
        a {
            display: inline-block;
            padding: 12px 24px;
            background-color: #98d7fb;
            color: #333;
            text-decoration: none;
            border-radius: 25px;
            font-family: sans-serif;
            font-size: 11pt;
        }
    </style>
</head>

<body>
    <script>
        // Get the URL from the path
        const url = "flet://flet-host/http%3A%2F%2F" + window.location.pathname.slice(1);

        // Create the button element
        const button = document.createElement('a');
        button.href = url;
        button.textContent = 'Open the project in Flet app';
        document.body.appendChild(button);

        // Set the meta refresh tag
        document.querySelector('head').innerHTML += `<meta http-equiv="refresh" content="0;url=${url}" />`;
    </script>
</body>

</html>
```

**Comment by Muddassir-Farooq-official** [comment]
Can you provide run able code example

**Comment by Muddassir-Farooq-official** [comment]
@FeodorFitsner ????

**Comment by Muddassir-Farooq-official** [comment]
> @FeodorFitsner ???? 

@FeodorFitsner @InesaFitsner kindly Clear this out because my project is stucked

**Comment by Muddassir-Farooq-official** [comment]
How to set up this

`<data android:path="/login/" /> <!-- path must start with "/product/" and then it can contain any number of characters; in this case, it would probably be a product ID. --> <data android:pathPattern="/product/.*" /> <!-- path must start with /account/, so it may be /account/confirm, /account/verify etc. --> <data android:pathPrefix="/account/" />`

**Comment by Muddassir-Farooq-official** [comment]
https://github.com/user-attachments/assets/02850880-a733-4a13-a834-f7fdb51529f9

@FeodorFitsner my problem is solved but I want to say that while writing in py projects.toml doesn't actually add intend filter and deep link configuration in mainfest.xml
